CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.4.1
+
+Release date: TBD
+
+### Bug Fixes
+
+#### macOS
```

Comment:
Since [v1.2.4](https://github.com/electron/electron/releases/tag/v1.2.4). [v1.2.3](https://github.com/electron/electron/releases/tag/v1.2.3) still uses "OS X".